pkg/rancher-desktop/window/index.ts

```diff
@@ -223,9 +223,9 @@ export function openExtension(id: string, relPath: string) {
         window, {
           ...CommandOrControl,
           shift: true,
-          key:   'i',
+          key:   'O',
```

Comment:
Could you add `// "capital-oh" to the right because right here it looks more like a `zero` than an `oh`?

Reply:
Added in 7f1335156e9c5d50c0ef5518341aaf6050153cc4